### PR TITLE
Minor cmd doc

### PR DESCRIPTION
**Pull request by andrewlayman**
Minor edits to cmd help docs.

### CODE REVIEW

**Comment by andrewlayman** [comment]
@lukehoban : Should I really update the CHANGELOG.MD?

File: cmd/refresh.go [hunk 1]
```diff
@@ -48,10 +48,8 @@ func newRefreshCmd() *cobra.Command {
 		Short: "Refresh the resources in a stack",
 		Long: "Refresh the resources in a stack.\n" +
 			"\n" +
-			"This command compares the current stack's resource state with the state known to exist in\n" +
-			"the actual cloud provider. Any such changes are adopted into the current stack. Note that if\n" +
-			"the program text isn't updated accordingly, subsequent updates may still appear to be out of\n" +
-			"synch with respect to the cloud provider's source of truth.\n" +
+			"This command resets the 'Pulumi.<stack name>.yaml' file to match the state snapshot copy from the last\n" +
+			"'pulumi up' command.\n" +
```
Comment (lukehoban) [inline cmd/refresh.go]: This doesn't looks correct.

File: cmd/state.go [hunk 2]
```diff
@@ -40,7 +40,7 @@ func newStateCmd() *cobra.Command {
 		Short: "Edit the current stack's state",
 		Long: `Edit the current stack's state
 
-Subcommands of this command can be used to surgically edit parts of a stack's state. These can be useful when
+Subcommands of this command can be used to surgically edit parts of a stack's deployment state. These can be useful when
```
Comment (lukehoban) [inline cmd/state.go]: This doesn't seem to be a net improvement, and I don't think is consistent with the typical way we refer to "state".

File: cmd/up.go [hunk 3]
```diff
@@ -269,10 +269,11 @@ func newUpCmd() *cobra.Command {
...
 			"is computed by running the current Pulumi program and observing all resource allocations to produce a\n" +
-			"resource graph. This goal state is then compared against the existing state to determine what create,\n" +
-			"read, update, and/or delete operations must take place to achieve the desired goal state, in the most\n" +
-			"minimally disruptive way. This command records a full transactional snapshot of the stack's new state\n" +
-			"afterwards so that the stack may be updated incrementally again later on.\n" +
+			"resource graph. This goal state is then compared against the existing state, as recorded in the\n" +
+			"snapshot taken during the last `pulumi up` command, to determine what create,read, update, and/or \n" +
+			"delete operations must take place to achieve the desired goal state, in the least disruptive way.\n" +
+			"This command records a full transactional snapshot of the stack's new state afterwards in the\n" +
+			"workspace backend so that the stack may be updated incrementally again later on.\n" +
```
Comment (lukehoban) [inline cmd/up.go]: Similarly not sure this is a net improvement.

File: cmd/refresh.go (hunk 1 above)
Comment (andrewlayman) [reply cmd/refresh.go]: I wrote this based on Patrick's input.  If it is not correct, then at least the current text is confusing and we should write here whatever is correct.  How about I circle back to Patrick again and check this?

File: cmd/state.go (hunk 2 above)
Comment (andrewlayman) [reply cmd/state.go]: I'm OK with not making this change.

File: cmd/up.go (hunk 3 above)
Comment (andrewlayman) [reply cmd/up.go]: I think it is a net improvement, as it is more clear about the scope.

**Pull request closed without merging**